### DISCUSSION TITLE
read error

### DISCUSSION THREAD

**Discussion by meganewing** [sr320/course-fish546-2021#101] [Q&A] [answered]
When I ran my slurm script on mox everything seemed to be going fine (I was just checking the md5sums for the files I had uploaded), but when it finished, I got an error saying `md5sum: /gscratch/srlab/mewing0/: read error` was hoping someone could shed some light on one this error means? 

Slurm script is [here](https://github.com/fish546-2021/Megan-project/blob/main/code/0225-slurm.sh)

**Comment by kubu4** [comment, marked as answer]
You didn't specify the file that you want `md5sum` to work with.

**Reply by kubu4** [reply]
Also, you won't need to (probably don't want to) specify the full path to the file, since you're already in the directory.

**Reply by meganewing** [reply]
Thank you!
